cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/NewObjectParsingStrategy.java

```diff
@@ -25,9 +30,31 @@
 
     @Override
     protected List<CfChange> doParse(Change change) {
+        List<CfChange> cfChanges = new LinkedList<>();
+
+        // since JaVers doesn't explicitly create a change object for the space devs when spec bean is a new object
+        // so creating it manually
+        if (change.getAffectedObject().get() instanceof SpecBean) {
+            List<String> spaceDevelopers = ((SpecBean) change.getAffectedObject().get()).getSpaceDevelopers();
+            if (spaceDevelopers != null) {
+                String propertyName = "spaceDevelopers";
```

Comment:
It would be nicer if the property name was stored in a constant field. Even better if we guarantee that the field name of the `SpecBean` fits the property name via our `ObjectPropertyValidation` class.